crunchy-bridge-for-analytics/README.md

```diff
@@ -0,0 +1,46 @@
+
+[Crunchy Bridge for Analytics](https://docs.crunchybridge.com/analytics) allows users to query parquet files on S3 directly.
+
+
+1) Setup the cluster:
+
+- Go to https://crunchybridge.com/
+- Click `Create Cluster` -> `Create Analytics Cluster`
+- Choose Region (`eu-central-1`) where `s3://clickhouse-public-datasets/hits_compatible/hits.parquet` is
+- Click `Analytics-256`
+- Click `Create Cluster`
```

Comment:
(Tried to reproduce using the steps but the UI wanted my credit card details first to charge me > 6.8k $.)

Reply:
Well, that's for full month:) The costs are accrued as a pro rata portion of the monthly value, on minute basis.

(And, you can also [suspend](https://docs.crunchybridge.com/how-to/suspend-resume) the cluster when not used.)